source/changelog/r124.inc

```diff
@@ -0,0 +1,120 @@
+* An issue where Preliminary Payment Demands on Enterprise Plan would exclude the subscription price in the total Amount calculation and only contain subscription 2 to _n_.
+* A case where charges would have invalid ledger marker, causing subsequent issues during write-off.
+* Allowance ledger marker would be ignored during write off.
+* A race condition causing significant delays when creating initial subscriptions without going through the order process.
+* **[Hotfix]** An issue where additional products would not be populated for subsequent steps in chained subscription plans from the order process.
+* **[Preview]** API route for exporting |BetalingsService| files has been changed so the path is unique in the API description.
+* **[Preview]** |MobilePay| is now correctly set as the source on payments when automatic payment generation occurs.
+
+Deprecated
+----------
+We call out the following deprecations, the endpoints/field will be removed in a future version, the exact timing depending on existing use and adoption of replacements.
+
+* We are deprecating the endpoints ``/paymentdemand/{id}/credit/completely`` and ``/paymentdemand/{id}/credit/partially`` in favor of ``/paymentdemand/{id}/credit`` endpoint.
+* We are deprecating the field ``TaxPercentage`` on the endpoint for creating Account Payment Demand transaction ``POST /paymentdemand/``, in favour of specifying a list of Tax Details.
+
+Merchant Client
+===============
+The following new features and improvements are now available in the merchant client.
+
+Added
+-----
+
+* Option to select Subscriber Account when creating payments manually.
+* Support for connecting payments to subscriber accounts if not matched automatically.
+* Option of assigning a `Name` to shared users instead of just displaying the email.
+* **[Preview]** OIO is now supported as a payment method for new agreements, similar to EHF.
+* **[Preview]** |MobilePay| support.
+
+    * Payments with source MobilePay gets a customized branding logo.
+    * MobilePay Agreements can be viewed with transactions.
+
+
+Changed
+-------
+
+* Additional validations for switching payment agreement on existing subscriptions, handling requirements for Organization Number if switching to `EHF` for instance.
+* Option to reset product selection during creation of a new subscription.
+* The |Vipps| Account Configuration via now also enables configuration of |MobilePay|, determined by the chosen country.
+* Tooltip shown for Organization Number to clarify different meanings depending on country of operation.
+* Number of available shared users now calculated using backend provided summary field on the subscription plan.
+
+Fixed
+-----
+
+* An issue where the username/principal name was shown for a user instead of the assigned Display Name.
+* An issue where searching for users would target the wrong underlying field.
+* Handling of missing responses from remote source on provider transactions. 
```

Comment:
Maybe describe a little bit better what this is about... Someone might not understand what a provider is.